framework/include/wifi_manager/wifi_manager.h

```diff
@@ -293,7 +315,7 @@ wifi_manager_result_e wifi_manager_connect_ap_config(wifi_manager_ap_config_s *c
  * @brief Connect to an access point.
  * @details @b #include <wifi_manager/wifi_manager.h>
  * @param[in] config ssid, passphrase, authentication type, and cryto type of the access point which the wi-fi interface connect to.
```

Comment:
Spell mistakes needs to be corrected
cryto => crypto

Reply:
Modification completed. Thanks.